### ISSUE TITLE
⌨ issue with form controls

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15600]
### ⌨ Accessibility and Expected Behaviour

🛡️Issues reported from Fable tester: 

1. in "employee verification" Link or button can't be activated by voice commands. The employee verification link cannot be selected using the command "click employee verification". I had to use the command "click button". 

2. when adding a new experience I cannot fill in the radio buttons or checkboxes using the name of the button. Example, "click government of Canada", I had to use the command "click radio button" or "click checkbox".
3.  adding work streams: I cannot select the work streams checkboxes by using the command "click" followed by the name of the Work Stream. I had to use the command "click checkbox".

**Expected behaviour:** User can activate form fields by using the name provided rather than having to say the technical name. 

### WCAG 2.1 Criteria

2.5.3 Label in Name 
2.1.1 Keyboard

### 📋 Steps to Reproduce

1. sign in 
2. Navigate to career experience
3. Add a new experience
4. Use voice commands on the form fields

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

Dragon Naturally Speaking

### ✅ Acceptance Criteria

- [ ] User can activate form fields by using the name provided rather than having to say the technical name.

**Comment by esizer** [comment]
Looks like this is not reproducible. I also had issues reproducing on macOS with voice control. We cann always reopen if it comes up again during testing.
